src/chttpd/src/chttpd_misc.erl

```diff
@@ -442,3 +486,25 @@ message_queues(Registered) ->
         {Type, Length} = process_info(whereis(Name), Type),
         {Name, Length}
     end, Registered).
+
+parse_bool_param(_, Val) when is_boolean(Val) ->
+    Val;
+parse_bool_param(_, "true") -> true;
+parse_bool_param(_, "false") -> false;
+parse_bool_param(Name, Val) ->
+    Msg = io_lib:format("Invalid value for ~s: ~p", [Name, Val]),
+    throw({query_parse_error, ?l2b(Msg)}).
+
+parse_all_dbs_param("info", Value) ->
+    [{info, parse_bool_param("info", Value)}];
+parse_all_dbs_param("keys", Value) ->
+    [{keys, ?JSON_DECODE(Value)}];
+parse_all_dbs_param(Key, Value) ->
+    [{extra, {Key, Value}}].
+
+validate_alldbs_query(info, Value, Args) ->
```

Comment:
should it be named `validate_all_dbs_query`? To be consistent with:
 -  `parse_all_dbs_param`
 - `handle_all_dbs_req`